client/client-multi/client-cpp/impl/utils/ThreadPool.cpp

```diff
@@ -114,32 +119,26 @@ void ThreadPool::shutdownNow()
 void ThreadPool::start()
 {
     for (std::size_t i = 0; i < workerCount_; ++i) {
-        workers_.push_back(std::thread(Worker(*this)));
+        workers_.emplace_back(Worker(*this));
     }
+
+    state_ = State::RUNNING;
```

Comment:
It starts only once
`        if (state_ == State::CREATED) {
            start();
        }
`